scripts/spot-grid.py

```diff
@@ -14,6 +14,7 @@
 import cairo
 import logging
 import numpy
+import skimage.io
```

Comment:
So far we don't depend on skimage. Let's try to not depend on it. If you need more than TIFF + HDF5, you could have a look at Pillow (which we already depend on).